src/util/Asio.h

```diff
@@ -0,0 +1,25 @@
+//   Copyright 2023, University of Freiburg,
+//   Chair of Algorithms and Data Structures.
+//   Author: Robin Textor-Falconi <textorr@informatik.uni-freiburg.de>
+
+#ifndef QLEVER_ASIO_H
+#define QLEVER_ASIO_H
+
+#include <boost/asio/awaitable.hpp>
```

Comment:
This file should have a somewhat longer name like `BoostAsioUtils` or `AsioHelpers` etc.